about.html

```diff
@@ -0,0 +1,158 @@
+  <div class="headline-bg">
+    <header>
+
+      <div>
+        <ul class="top-header-list">
+          <li><img src="./Images/facebook.png" alt="facebook icon"></li>
+          <li><img src="./Images/twitter.png" alt="twitter icon"></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li class="last"><a href="#">Log out</a></li>
+        </ul>
+      </div>
+
+      <nav>
+        <!-- Desktop Navbar -->
+        <div class="desktop-nav">
+          <img class="page-logo" src="./Images/logo.png" alt="Page Logo">
+          <ul class="desktop-nav-list">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#services">Our Services</a></li>
+            <li><a href="">Our Team</a></li>
+            <li><a href="#supporters">Sponsors</a></li>
+            <li class="last"><a class="list-btn" href="#">Our Projects</a></li>
+          </ul>
+        </div>
+
+        <!-- Mobile Navbar -->
+        <div>
+          <img id="hamburger-icon" class="union-icon" src="./Images/hamburger.png" alt="Hamburger icon">
+
+          <!-- close icon-->
+          <a href="#" class="close-icon hidden">
+            <img src="./Images/close.png" alt="close icon">
+          </a>
+
+          <!-- mobile menu -->
+          <ul class="mobile-menu hidden">
+            <li class="mobile-list"><a href="./index.html">Home</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#services">Our Services</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#contact-me">Our Team</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#supporters">Sponsors</a></li>
+            <hr>
+            <li class="mobile-list"><a href="#">Our Projects</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="about-headline">
+      <article class="about-heading">
+        <h3 class="about-title">Hello Front-End Lovers!</h3>
+        <h1 class="main-heading">Front-End School, best place to learn frontend technologies!</h1>
+      </article>
+
+      <article class="about-intro">
+
+        <p class="about-info">
+          Welcome to our front-end web developer learning pathway!
+          Here we provide you with a structured pathway that will teach you all you need to know to become a front-end
+          web developer. Work through each section, learning new skills (or improving existing ones) as you go along.
+          Each section includes exercises and assessments to test your understanding before you move forward. <br>
+          Choosing the right resources is the key to becoming a successful developer. Most of the beginners follow the
+          wrong resources, and it reults in disappointment. We have chosen best out of best resources available online.
+        </p>
+
+        <p class="reach-us">frontendschool23@gmail.com</p>
+
+      </article>
+    </section>
```

Comment:
Good job! but please make sure to use the same spacing for your layout as it is in the original design, additionally, you have an extra `h3` about the title, which is not part of the original design, please make sure to provide a similar layout to the original design.

Orignal:
![image](https://user-images.githubusercontent.com/34779928/205145501-61fa02df-6883-48c1-aa0e-dba04237493a.png)


Yours:
![image](https://user-images.githubusercontent.com/34779928/205145454-1513c94e-9143-444f-aaa3-2e3a20b7abe8.png)